listener/upload.go

```diff
@@ -104,15 +104,15 @@ func HandleUpload(event HostEvent) error {
 	if sys.UnchangedSince != nil && sys.LastEvaluation != nil {
 		if sys.UnchangedSince.Before(*sys.LastEvaluation) {
 			messagesReceivedCnt.WithLabelValues(EventUpload, ReceivedSuccessNoEval).Inc()
-			utils.Log("inventoryID", event.Host.ID).Debug(UploadSuccessNoEval)
+			utils.Log("inventoryID", event.Host.ID).Info(UploadSuccessNoEval)
```

Comment:
Is that all? Only listener updates? Don't we need to update anything in other components?